geronimo-json_1.0_spec/src/main/java/javax/json/spi/JsonProvider.java

```diff
@@ -28,6 +28,9 @@
 import javax.json.stream.JsonGeneratorFactory;
 import javax.json.stream.JsonParser;
 import javax.json.stream.JsonParserFactory;
+
+import org.apache.geronimo.osgi.locator.ProviderLocator;
```

Comment:
then please ensure it is shaded in another package. issue is several spec jars have all the same classes and it can lead to issues in some classloading environments.